css/theme-fire.css

```diff
@@ -72,6 +72,7 @@ ul.bullets {
 }
 .main-container {
   clear: both;
+  padding-top: 64px;
```

Comment:
This is causing a white bar of 64px 

<img width="838" alt="screen shot 2017-04-03 at 3 49 55 pm" src="https://cloud.githubusercontent.com/assets/1327050/24605136/39363bce-1885-11e7-9a83-0d41bb490e18.png">